gslib/tests/test_temporary_file_util.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for temporary_file_util.py."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+from gslib import storage_url
+from gslib.tests import testcase
+from gslib.utils import temporary_file_util
+
+
+class TestTemporaryUtil(testcase.GsUtilUnitTestCase):
+  """Test temporary file utils."""
+
+  def test_gets_temporary_file_name(self):
```

Comment:
I think gsutil tends to use lower camel case for test names. I don't think it's 100% consistent but may be good to change this to match the other test file.

Reply:
Yes, great catch.